src/pages/console/uibuilder/eventhandling.mdx

```diff
@@ -3,7 +3,7 @@ export const meta = {
   description: `Figma to React code with Amplify Studio`,
 };
 
-Amplify Studio offers the ability to bind UI component events (`onClick`, `onChange`, and more) to actions to build interactive components. Use the UI component editor to map UI components' events to actions for navigation, data manipulation, authentication, and more. All data bindings get automatically included in generated code.
+[Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app) offers the ability to bind UI component events (`onClick`, `onChange`, and more) to actions to build interactive components. Use the UI component editor to map UI components' events to actions for navigation, data manipulation, authentication, and more. All data bindings get automatically included in generated code.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside Studio. Like right above "select an UI..." have "From Amplify Studio:"